glean-core/rlb/src/lib.rs

```diff
@@ -541,5 +576,94 @@ pub fn test_reset_glean(cfg: Configuration, client_info: ClientInfoMetrics, clea
     initialize(cfg, client_info);
 }
 
+/// Sets a debug view tag.
+///
+/// When the debug view tag is set, pings are sent with a `X-Debug-ID` header with the
+/// value of the tag and are sent to the ["Ping Debug Viewer"](https://mozilla.github.io/glean/book/dev/core/internal/debug-pings.html).
+///
+/// # Arguments
+///
+/// * `tag` - A valid HTTP header value. Must match the regex: "[a-zA-Z0-9-]{1,20}".
+///
+/// # Returns
+///
+/// This will return `false` in case `tag` is not a valid tag and `true` otherwise
+/// or if the tag is set before Glean is initialized.
```

Comment:
I find this sentence a bit ... harder to read than necessary.
What about changing it to cover the _exceptional_ case more clearly:

```suggestion
/// This will return `false` in case `tag` is not a valid tag and `true` otherwise.
/// If called before Glean is initialized it will always return `true`.
```